readme.md

```diff
@@ -1,4 +1,4 @@
-PAWS
+#PAWS
 
 
 A Web Shell (PAWS) is a Jupyter notebooks deployment that has been customized to make interacting with Wikimedia wikis easier. It allows users to create and share documents that contain live code, visualizations such as graphs, rich text, etc. The user created notebooks are a powerful tool that enables data analysis and scientific research, and also transforms the way in which programmers write code - by enabling an exploratory environment with a quick feedback loop, and a low barrier for entry through it's easy to use graphical interface. 
```

Comment:
It would be more correct here to say "PAWS: A Web Shell (PAWS)" at the start of the first sentence, it's a recursive acronym like GNU.

Reply:
okay, thanks for reviewing.
Now could u review it again, i just completed your suggested changes. @crookedstorm